### PR TITLE
Reload Bug when leaving meeting with Firefox Browser

### PR DESCRIPTION
**Pull request by birkastoana**
Changed the bbb-autoclose.html to fix the reload-bug wen leaving a meeting with firefox.

### CODE REVIEW

File: tool/src/webapp/bbb-autoclose.html [hunk 1]
```diff
@@ -26,23 +26,23 @@
...
     window.close();
 }
 </script>
 </head>
-<body style="width:99.5%">
-    <span style="margin:50px auto">
-        <a href="javascript:;" onclick="closeWindow();closeWindowAlternative();">Close this window</a>
+<body onload="closeWindow()" style="width:99.5%">
+    <span style="margin:50px auto; font-size: x-large">
+        <p>You left the meeting</p>
```
Comment (jfederico) [inline tool/src/webapp/bbb-autoclose.html:43]: The strings need to be localized

File: tool/src/webapp/bbb-autoclose.html [hunk 2]
```diff
@@ -26,23 +26,23 @@
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
 <script type="text/javascript" language="JavaScript" src="/bbb-tool/lib/jquery-1.3.2.min.js"></script> 
 <script type="text/javascript" language="JavaScript">
-$(document).ready(function(){
-    closeWindow(); closeWindowAlternative();
-});
+
```
Comment (jfederico) [inline tool/src/webapp/bbb-autoclose.html:29]: Can you remove this extra line?

File: tool/src/webapp/bbb-autoclose.html [hunk 3]
```diff
@@ -26,23 +26,23 @@
...
 
 function closeWindow() {
-    var objWindow = window.open(location.href, "_self");
-    objWindow.close(); 
+    var myWindow = window.open('', '_self', '');
```
Comment (jfederico) [inline tool/src/webapp/bbb-autoclose.html:32]: Can you restore the name for the variable. One line less to be changed in this PR.

**Review by jfederico — CHANGES_REQUESTED** [review]

**Comment by jfederico** [comment]
Thanks for the PR @birkastoana

File: tool/src/webapp/bbb-autoclose.html (hunk 1 above)
Comment (birkastoana) [reply tool/src/webapp/bbb-autoclose.html:43]: I'm not quite sure how to implement that. Where do you tell bbb to redirect to that static html?

**Comment by birkastoana** [comment]
Will make a new PR

**Pull request closed without merging**